src/main/java/org/mitre/synthea/world/agents/behaviors/planeligibility/QualifyingAttributesEligibility.java

```diff
@@ -0,0 +1,154 @@
+package org.mitre.synthea.world.agents.behaviors.planeligibility;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.lang3.math.NumberUtils;
+import org.mitre.synthea.helpers.SimpleCSV;
+import org.mitre.synthea.helpers.Utilities;
+import org.mitre.synthea.world.agents.Person;
+
+/**
+ * An eligibility criteria based on whether a person has the given attributes.
+ */
+public class QualifyingAttributesEligibility implements IPlanEligibility {
+
+  private final List<AttributeQualifier> qualifyingAttributes;
+
+  /**
+   * Constructor.
+   * @param attributeInput  The "|" delimited string or file of attribute logics.
+   */
+  public QualifyingAttributesEligibility(String attributeInput) {
+    if (attributeInput.contains("/")) {
+      // The input is a file, so we have a file that defines the eligible conditions.
+      qualifyingAttributes = buildQualifyingAttributesFile(attributeInput);
+    } else {
+      // The input is a set of attributes.
+      qualifyingAttributes = new ArrayList<>();
+      for (String attributeExpression : Arrays.asList(attributeInput.split("\\|"))) {
+        qualifyingAttributes.add(covertAttributeExpressionToLogic(attributeExpression));
+      }
+    }
+  }
+
+  @Override
+  public boolean isPersonEligible(Person person, long time) {
+    return qualifyingAttributes.stream().anyMatch(attributeLogic ->
+      attributeLogic.checkLogic(person));
+  }
+
+  /**
+   * Builds a list of attributes that would qualify a person for this eligibility type.
+   * @return  A list of qualifying attributes.
+   */
+  private static List<AttributeQualifier> buildQualifyingAttributesFile(String fileName) {
+    String resource = null;
+    Iterator<? extends Map<String, String>> csv = null;
+    try {
+      resource = Utilities.readResource(fileName);
+      csv = SimpleCSV.parseLineByLine(resource);
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+
+    List<AttributeQualifier> attributeEligibilities = new ArrayList<AttributeQualifier>();
+    while (csv.hasNext()) {
+      Map<String, String> row = csv.next();
+      String attributeRow = row.get("attributes");
+      String[] attributes = attributeRow.split("\\|");
+      for (String attributeExpression : attributes) {
+        AttributeQualifier attributeLogic = covertAttributeExpressionToLogic(attributeExpression);
+        attributeEligibilities.add(attributeLogic);
+      }
+    }
+    return attributeEligibilities;
+  }
+
+  private static AttributeQualifier covertAttributeExpressionToLogic(String attributeExpression) {
+    String regex = " ";
+    String[] splitExpression = attributeExpression.split(regex);
```

Comment:
This is another place to think about the user experience. The expression has to be exactly attribute, one space character, comparator, one space character and then value.

Users may expect to be able to omit the whitespace, or include more whitespace without it being significant. Consider a more robust regex with match groups to allow more variation in input.